src/TConsole.cpp

```diff
@@ -346,17 +346,20 @@ TConsole::TConsole(Host* pH, bool isDebugConsole, QWidget* parent)
     layoutLayer2->setMargin(0);
     layoutLayer2->setSpacing(0);
 
-    auto buttonMainLayer = new QWidget; //( layerCommandLine );
+    buttonMainLayer = new QWidget; //( layerCommandLine );
+    buttonMainLayer->setObjectName(QStringLiteral("buttonMainLayer"));
     buttonMainLayer->setSizePolicy(sizePolicy);
     buttonMainLayer->setContentsMargins(0, 0, 0, 0);
     auto layoutButtonMainLayer = new QVBoxLayout(buttonMainLayer);
+    layoutButtonMainLayer->setObjectName(QStringLiteral("layoutButtonMainLayer"));
     layoutButtonMainLayer->setMargin(0);
     layoutButtonMainLayer->setContentsMargins(0, 0, 0, 0);
 
     layoutButtonMainLayer->setSpacing(0);
     /*buttonMainLayer->setMinimumHeight(31);
            buttonMainLayer->setMaximumHeight(31);*/
     auto buttonLayer = new QWidget;
+    buttonLayer->setObjectName(QStringLiteral("buttonLayer"));
     auto layoutButtonLayer = new QGridLayout(buttonLayer);
```

Comment:
It can be useful (in GammaRay for instance) to give an object name to layouts as well!

Reply:
Done.